app/controllers/api/v1/requests_controller.rb

```diff
@@ -51,7 +51,7 @@ def rbac_scope(relation)
         # for admin
         return relation unless ids
 
-        Rails.logger.info("Accessible #{relation.model.table_name} ids: #{ids}")
+        Rails.logger.debug { "Accessible #{relation.model.table_name} ids: #{ids}" }
```

Comment:
This is the log line I saw posting 9400+ IDs.  That will use up some time to write out.  Hoping this helps.

Reply:
Then I will remove this message, just in the future real case has the same scenario.